Gopkg.toml

```diff
@@ -109,7 +109,7 @@ ignored = [
 
 [[constraint]]
   name = "github.com/envoyproxy/go-control-plane"
-  branch = "revert_grpc_version_update"
+  revision = "c17582cc6514d173e4309d4e3bb16a150efcfcf7"
```

Comment:
? where is the associated change in the vendor file?

Reply:
The associated changes in vendor have been added.